storage/ethereum.js

```diff
@@ -0,0 +1,32 @@
+const Web3 = require('web3');
+const TruffleContract = require('truffle-contract');
+const storeSol = require('../build/contracts/Store.json');
+
+const web3 = new Web3(new Web3.providers.HttpProvider('http://localhost:9545'));
```

Comment:
Can we put this into a config object so we can change it by the environment

Reply:
yes, we should. i will create a separate file for web3.